docs/dictionary/property/recordInput.lcdoc

```diff
@@ -64,5 +65,6 @@ bit builds of LiveCode.
 
```

Comment:
Is it possible to put an alternative here or specifically state that there is no way to set a recordInput from LCv8.1 just to make it absolutely clear? (We may have to go through all references to QT stuff later and do the same).